scripts/zephyr_module.py

```diff
@@ -394,9 +396,12 @@ def west_projects():
     return None
 
 
-def parse_modules(zephyr_base, modules=None, extra_modules=None):
+def parse_modules(zephyr_base, manifest=None, modules=None, extra_modules=None):
```

Comment:
Should it be forbidden to set both `manifest` and `modules` at the same time ?
Currently providing `modules` will always prevent the use of `manifest` and effectively make it void.

Reply:
I think it's OK that we allow all combos given that some will take precedence over others.

Reply:
ok